pallets/collator-selection/src/lib.rs

```diff
@@ -499,29 +512,86 @@ pub mod pallet {
 			collators.extend(candidates.into_iter().collect::<Vec<_>>());
 			collators
 		}
-		/// Kicks out and candidates that did not produce a block in the kick threshold.
+
+		/// Removes collators with unsatisfactory performance
+		/// Returns the removed AccountIds
 		pub fn kick_stale_candidates(
 			candidates: Vec<CandidateInfo<T::AccountId, BalanceOf<T>>>,
 		) -> Vec<T::AccountId> {
-			let now = frame_system::Pallet::<T>::block_number();
-			let kick_threshold = T::KickThreshold::get();
-			candidates
+			use num_traits::float::FloatCore; // For no_std compatible f64::ceil()
```

Comment:
This import is needed to pull the f64::ceil() fn into scope but I am getting a

```
warning: unused import: `num_traits::float::FloatCore`
   --> pallets/collator-selection/src/lib.rs:521:8
    |
521 |             use num_traits::float::FloatCore; 
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default
```

any hints on how to squelch that?

Reply:
resolved by using libm::ceil() and f64